src/api/layers/ExternalGroupOfLayers.class.js

```diff
@@ -35,32 +35,32 @@ export default class ExternalGroupOfLayers extends ExternalLayer {
      * @param {boolean} isLoading Set to true if some parts of the layer (e.g. metadata) are still
      *   loading
      */
-    constructor(
-        name,
-        opacity,
-        visible,
-        baseUrl,
-        layerId,
+    constructor({
```

Comment:
Name and base url should not have default values that too risky they are Mandatory parameter

Reply:
I thought I should maybe validate the inputs and raise exception if what was given isn't valid. This way I can keep the "optional" approach, but check that we have sufficient information to build the instance in the constructor.
What is your take on that? Good or bad idea?